drivers/adc/ad7124/ad7124.c

```diff
@@ -8,7 +7,8 @@
 *
 * All rights reserved.
 *
-* Redistribution and use in source and binary forms, with or without modification,
+* Redistribution and use in source and binary forms, with
```

Comment:
squash this with previous commit since they are both astyle modifications.

Reply:
will do